packages/cli/test/utils/simulation/beacon_clients/lodestar.ts

```diff
@@ -96,7 +95,7 @@ export const generateLodestarBeaconNode: BeaconNodeGenerator<BeaconClient.Lodest
       },
       health: async () => {
         try {
-          await got.get(`http://${address}:${ports.beacon.httpPort}/eth/v1/node/health`);
+          await fetch(`http://${address}:${ports.beacon.httpPort}/eth/v1/node/health`);
```

Comment:
I don't think this is correct, please check how `got` was handling http errors and if it was throwing that error. Native fetch will only throw if there is a network error, and otherwise if the server returns a >400 status code it will return `res.ok=false`.

I think what we can just do here is just simply return

```suggestion
          return await fetch(`http://${address}:${ports.beacon.httpPort}/eth/v1/node/health`);
```

Reply:
@nflaig 
Can I delete here?
```
return {ok: true};
```